<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ThreadControl</class>
 <widget class="QWidget" name="ThreadControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>616</width>
    <height>252</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QRadioButton {
	color: rgb(255, 255, 255);
}

QLabel {
	color: rgb(255, 255, 255);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="title">
      <string>Core </string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="topMargin">
       <number>30</number>
      </property>
      <item>
       <widget class="QRadioButton" name="radioButton_coreEnabled">
        <property name="text">
         <string>Enabled</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="leftMargin">
         <number>30</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_scalingGovernor">
          <property name="text">
           <string>Governor: </string>
          </property>
          <property name="margin">
           <number>10</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_scalingGovernor_info">
          <property name="text">
           <string/>
          </property>
          <property name="margin">
           <number>10</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
